src/common/BaseAnnotator.ts

```diff
@@ -91,6 +100,15 @@ export default class BaseAnnotator {
         // Called by box-content-preview
     }
 
+    setActiveAnnotationById = (annotationId: string): void => {
```

Comment:
Is this name accurate? We're setting the active annotation id, not the active annotation _by_ the id. How about just `setActiveAnnotationId`?

Reply:
Sounds good